link/src/main/java/com/stripe/android/link/ui/Common.kt

```diff
@@ -19,16 +21,30 @@ typealias BottomSheetContent = @Composable ColumnScope.() -> Unit
 
 @Composable
 internal fun ScrollableTopLevelColumn(
+    omitTopPadding: Boolean = false,
     content: @Composable ColumnScope.() -> Unit
 ) {
+    val padding = remember(omitTopPadding) {
```

Comment:
Added this, because the spacing at the top of the screen looks fairly large now in the sub-screens.